### PR TITLE
[Doc] Clearer explanation for some props in AutompleteInput

### PR DESCRIPTION
**Pull request by adguernier**
Problem:
- A sentence was not clear in my opinion
- What does `create` prop ?

Solution:
- Sentence rearrangement
- Add a video to show what `create` prop does

### CODE REVIEW

File: docs/AutocompleteInput.md [hunk 1]
```diff
@@ -391,7 +397,7 @@ import { AutocompleteInput, ReferenceInput } from 'react-admin';
 </ReferenceInput>
 ```
 
-In that case, react-admin uses the [`recordRepresentation`](./Resource.md#recordrepresentation) of the related resource to display the record label. In the example above, `<AutocompleteInput>` uses the resource representation of the `authors` resource, which is the `name` property.
```
Comment (fzaninotto) [inline docs/AutocompleteInput.md:394]: I don't think you should remove this sentence. If it's hard to understand, perhaps you need to add a snippet about the `<Resource>` definition?

File: docs/img/autocomplete-input-with-create.mp4
Comment (fzaninotto) [inline docs/img/autocomplete-input-with-create.mp4:1]: Could you make a wider screencast (it's a bit narrow), and go beyond clicking on the "create" option to show the entire option creation process?

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: docs/AutocompleteInput.md [hunk 2]
```diff
@@ -384,6 +390,14 @@ const optionRenderer = choice => `${choice.first_name} ${choice.last_name}`;
 `optionText` is also useful when the choices are records [fetched from another resource](#fetching-choices), and `<AutocompleteInput>` is a child of a [`<ReferenceInput>`](./ReferenceInput.md). 
 
 ```jsx
+// in ./index.js
+import {Admin, Resource } from 'react-admin';
+
+<Admin {...adminProps}>
+    <Resource name="authors" recordRepresentation={record => record.name} />
+</Admin>
```
Comment (fzaninotto) [inline docs/AutocompleteInput.md:398]: That's not valid React!

```suggestion
import { Admin, Resource, ListGuesser } from 'react-admin';
import { dataProvider } from './dataProvider';
import { PostCreate } from './PostCreate';

export const App = () => (
    <Admin dataProvider={dataProvider}>
        <Resource name="posts" list={ListGuesser} create={PostCreate} />
        <Resource name="authors" recordRepresentation={record => record.name} />
    </Admin>
    );
```

Also, I would put that snippet *after* the PostCreate snippet.

File: docs/AutocompleteInput.md [hunk 3]
```diff
@@ -384,6 +390,14 @@ const optionRenderer = choice => `${choice.first_name} ${choice.last_name}`;
...
+
+// in ./PostCreate.jsx
 import { AutocompleteInput, ReferenceInput } from 'react-admin';
 
 <ReferenceInput label="Author" source="author_id" reference="authors">
```
Comment (fzaninotto) [inline docs/AutocompleteInput.md:403]: You need to make it a component

**Review by fzaninotto — APPROVED** [review]

**Review by djhi — APPROVED** [review]

File: docs/AutocompleteInput.md [hunk 4]
```diff
@@ -411,7 +411,7 @@ import { PostCreate } from './PostCreate';
 export const App = () => (
     <Admin dataProvider={dataProvider}>
         <Resource name="posts" list={ListGuesser} create={PostCreate} />
-        <Resource name="authors" recordRepresentation="name" />
+        <Resource name="authors" recordRepresentation="last_name" />
```
Comment (fzaninotto) [inline docs/AutocompleteInput.md:414]: I used `last_name` in this example because `name` is the default, so it makes no sense to set it.

**Comment by fzaninotto** [comment]
The video is too wide and too tall, and it's barely readable.

**Comment by adguernier** [comment]
> The video is too wide and too tall, and it's barely readable.

I update videos, let me know if their are ok :)

**Review by fzaninotto — APPROVED** [review]

File: docs/AutocompleteInput.md [hunk 5]
```diff
@@ -799,7 +831,7 @@ const CreateCategory = () => {
 
 **Tip:** As showcased in this example, react-admin provides a convenient hook for accessing the filter the user has already input in the `<AutocompleteInput>`: `useCreateSuggestionContext`.
 
-The `Create %{item}` option will only be displayed once the user has already set a filter (by typing in some input). If you expect your users to create new items often, you can make this more user-friendly by adding a placeholder text like this:
+The `Create %{item}%` option will only be displayed once the user has already set a filter (by typing in some input). If you expect your users to create new items often, you can make this more user-friendly by adding a placeholder text like this:
```
Comment (slax57) [inline docs/AutocompleteInput.md:834]: Actually this was not a typo, this is the way we declare template variables in polyglot :grimacing: 
(see https://github.com/marmelab/react-admin/blob/master/packages/ra-language-english/src/index.ts#L16)

**Pull request merged**